include/fastdds-statistics-backend/types/JSONTags.h

```diff
@@ -183,6 +183,14 @@ constexpr const char* DATA_VALUE_REMOTE_ENTITY_TAG  = "remote_id";
 //! Key tag for discovered (bool) value in data
 constexpr const char* DATA_VALUE_DISCOVERED_TAG     = "discovered";
 
+/////
+// StatisticsBackend info
+
+//! Key tag for the ID of an entity
+constexpr const char* ID_INFO_TAG                   = "id";
+//! Key tag for the EntityKind of an entity
+constexpr const char* KIND_INFO_TAG                 = "kind";
```

Comment:
[Documentation](https://fast-dds-statistics-backend.readthedocs.io/en/latest/rst/statistics_backend/get_info.html) should be updated and also the examples if we extend the return information.

I think it is a good idea to provide the `EntityKind` because the user does not have to know of which kind is the entity he is getting the info from. But maybe the `EntityId` is not needed because is the parameter that the user passes to the method. Have you talked with Iker about this?

Reply:
Yes, I talked with Iker about this, maybe I do not remember exactly the reason to write the ID, but i think it was that if the user use the info of the entity later on and not when he calls get_info(), would be useful for him to have the ID stored.

Reply:
Documentation updated

Reply:
Totally true the reason Iker gave you. I agree